.github/workflows/release.yaml

```diff
@@ -0,0 +1,43 @@
+# Release workflow: Pushing a tag triggers this workflow,
+# which uses the name of the tag as the container tag/version
+# that goes into GHCR.
+name: Release
+
+# Build and publish Docker image upon pushing a tag
+on:
+  push:
+    tags:
+      - "v*"
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        # actions/checkout@v2 breaks annotated tags by converting them into
+        # lightweight tags, so we need to force fetch the tag again
+        # See: https://github.com/actions/checkout/issues/290
+      - name: "Repair tag"
+        run: git fetch -f origin ${{ github.ref }}:${{ github.ref }}
+      - name: "Verify tag is annotated"
+        run: if test x$(git for-each-ref ${{ github.ref }} | awk '{print $2}') = xtag; then /bin/true; else echo "\"${{ github.ref }}\" does not look like an annotated tag!"; /bin/false; fi
+      - name: "Log in to Docker"
+        uses: docker/login-action@v1
+        with:
+          password: "${{ secrets.GITHUB_TOKEN }}"
+          registry: ghcr.io
+          username: "${{ github.actor }}"
+      - id: docker_build
+        name: "Build the release Docker image"
+        uses: docker/build-push-action@v2
+        with:
+          push: false
+          tags: "${{ github.repository }}:${{ github.ref }}"
+      - name: "Tag the build for GHCR"
```

Comment:
In main.yaml, there is a section that echos the values of the tags.  (The section titled "set TAG & TESTTAG for main/master").  Please add something similar to release.yaml.

Reply:
Done.